app/controllers/consultations_controller.rb

```diff
@@ -3,16 +3,4 @@ def index
     filter_params = params.except(:controller, :action, :format, :_)
     redirect_to publications_path(filter_params.merge(publication_filter_option: 'consultations'))
```

Comment:
It's a shame to have to leave this in. We could just add the redirect into the router. That might be a bit 'magic' though. Thoughts?